lib/FileUploader/FileUploader.js

```diff
@@ -107,5 +110,6 @@ FileUploader.propTypes = {
 
 FileUploader.defaultProps = {
   multiple: true,
+  disabled: false,
```

Comment:
Do you really need this? If disabled prop isn't provided it would work the same way

Reply:
Yes i know, but we will have a message from PropTypes that disabled prop isn't provided. It's not a required prop.

Reply:
If it isn't required prop, why do you need to have it by default?